### PR TITLE
trace-agent: reset metric counts correctly

### PR DESCRIPTION
**Pull request by djova**
### What does this PR do?

Fix wrong resetting of some trace agent metric counts. 

Reset should **SET** to zero, not **ADD** zero. :facepalm:

### Motivation

Have correct metrics.

### CODE REVIEW

File: releasenotes/notes/trace-normalization-metric-fix-count-a9decee155c70926.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Fix endlessly growing `traces_dropped` and `spans_malformed` metric counts.
```
Comment (gbbr) [inline releasenotes/notes/trace-normalization-metric-fix-count-a9decee155c70926.yaml]: Prefix this with “APM: “ and maybe phrase it better as “APM: Fixed a provlem where ... and ... where showing incorrect metrics.”

**Review by gbbr — COMMENTED** [review]
Dang! Not sure how I missed it. Can you add the milestone too? I guess 6.14.0 :-/

**Comment by djova** [comment]
> Dang! Not sure how I missed it. Can you add the milestone too? I guess 6.14.0 :-/

Added, not sure if we do bug fix releases. But if we do, would be good to get it out sooner.

File: releasenotes/notes/trace-normalization-metric-fix-count-a9decee155c70926.yaml (hunk 1 above)
Comment (djova) [reply releasenotes/notes/trace-normalization-metric-fix-count-a9decee155c70926.yaml]: Updated

**Review by gbbr — APPROVED** [review]
Thanks Dusan.

**Pull request merged**